images/api-server-with-data-mt/config.yaml

```diff
@@ -1,3 +1,5 @@
+repo:
+  repoUrl: file:///opt/kamu/workspace/.kamu/datasets
 auth:
   providers:
     # Dummy accounts for owners of example datasets
```

Comment:
Files like this require an update after merge of kamu-cli#540.
My proposal:
 - remove all `accountId` properties, current format expects a DID string, but for predefined accounts those should be auto-generated from names
 - `accountType: User` is a default, we can leave an override only for `Organization`.
 - `displayName` only needs to be specified, if it is different from `accountName`